engine/utils/Logging.h

```diff
@@ -68,6 +86,9 @@ class VariantContainer
     DEFINE_VARIANT_TYPE(const float& data, String::FromFloat(data));
     DEFINE_VARIANT_TYPE(const double& data, String::FromDouble(data));
     DEFINE_VARIANT_TYPE(const long& data, String::FromLong(data));
+    DEFINE_VARIANT_TYPE(const unsigned int& data, String::FromU32(data));
+    DEFINE_VARIANT_TYPE(const unsigned long& data, String::FromU32(data));
+    DEFINE_VARIANT_TYPE(const unsigned long long& data, String::FromU64(data));
```

Comment:
Just use `FromFormat` to get these three for now.

Reply:
I'm not sure what you're talking about here.